src/screencast/wlr_screencast.c

```diff
@@ -19,14 +19,26 @@
 #include "xdpw.h"
 #include "logger.h"
 
+static void wlr_frame_buffer_destroy(struct xdpw_screencast_instance *cast) {
+	// This check isn't needed
```

Comment:
If this check isn't necessary, you can just remove it. You can leave a comment below explaining that wl_buffer_destroy will not work on NULL if you like.

Reply:
Ok, I've changed it. Thanks.